functional-programming/FP.md

```diff
@@ -4,10 +4,13 @@
 30-45 minutes
 
 ### Prerequisites
-
+JavaScript basics
+`for` loops
+arrays
+Object-oriented programming
```

Comment:
FYI, object oriented programming is covered in this lesson, so no need to go into detail about it here:
https://github.com/Techtonica/curriculum/blob/master/javascript/javascript-7-oop.md

Reply:
no problem, I just wanted to make sure you didn't do more work than you had to!